xdg/globalkeyshortcuts.conf

```diff
@@ -40,7 +40,12 @@ Comment=screen shot
 Enabled=true
 Exec=screengrab, -f
 
-[Control%2BAlt%2BL.30]
+[Control%2BAlt%2BL.9]
```

Comment:
what does the .30 / .9 mean?

Reply:
As I understand this number is a sequential number. So I changed that.

Reply:
It's called "id" in the GUI and sorts the shortcuts. "30" roughly means "the last", in case other shortcuts are added (which are, by panel and runner); so, I think it's better not to change it. The next shortcut can have "31" as its id.

Reply:
OK @tsujan your comment make sense. I will revert that change and add the new shortcut with the id 31.

Reply:
@ACamposPT Updating this PR would be just fine.